Source/Scene/QuadtreePrimitive.js

```diff
@@ -435,20 +439,44 @@ define([
             customDataRemoved.length = 0;
         }
 
+        // Our goal with load ordering is to first load all of the tiles we need to 
+        // render the current scene at full detail.  Loading any other tiles is just
+        // a form of prefetching, and we need not do it at all (other concerns aside).  This
+        // simple and obvious statement gets more complicated when we realize that, because
+        // we don't have bounding volumes for the entire terrain tile pyramid, we don't
+        // precisely know which tiles we need to render the scene at full detail, until we do
+        // some loading.
+        //
+        // So our load priority is (from high to low):
+        // 1. Tiles that we _would_ render, except that they're not sufficiently loaded yet.
+        //    Ideally this would only include tiles that we've already determined to be visible,
+        //    but since we don't have reliable visibility information until a tile is loaded,
+        //    and because we (currently) must have all children in a quad renderable before we
+        //    can refine, this pretty much means tiles we'd like to refine to, regardless of
+        //    visibility. (high)
+        // 2. Tiles that we're rendering. (medium)
```

Comment:
It might just be terminology, but, for my understanding, why are "tiles that we're rendering" in the load order at all?  Aren't they already loaded?